app/models/job_invocation.rb

```diff
@@ -72,6 +72,10 @@ class JobInvocation < ApplicationRecord
 
   encrypts :password, :key_passphrase, :sudo_password
 
+  class Jail < Safemode::Jail
+    allow :task_id, :sub_task_for_host, :template_invocations_hosts
```

Comment:
We are not using the `task_id` anymore in the template, right?
```suggestion
    allow :sub_task_for_host, :template_invocations_hosts
```